### DISCUSSION TITLE
I can't find the Active Directory option in the IDM connector for version 6.1.1. And when I select the LDAP connector, the authentication succeeds, but no user is created in the IDM

### DISCUSSION THREAD

**Discussion by caoguozhen** [OpenIdentityPlatform/OpenIDM#60] [Q&A] [answered]
![image](https://github.com/user-attachments/assets/6fddde81-730a-4e3f-9457-6f6ce7c72792)

**Comment by vharseko** [comment, marked as answer]
[LDAP Connector](https://github.com/OpenIdentityPlatform/OpenICF/wiki/LDAP-Connector) has samples connect to AD over LDAP
* [provisioner.openicf-adldap.json](https://github.com/OpenIdentityPlatform/OpenIDM/blob/master/openidm-zip/src/main/resources/samples/provisioners/provisioner.openicf-adldap.json) provides a sample LDAP connector configuration for an Active Directory server.
* [provisioner.openicf-adldsldap.json](https://github.com/OpenIdentityPlatform/OpenIDM/blob/master/openidm-zip/src/main/resources/samples/provisioners/provisioner.openicf-adldsldap.json) provides a sample LDAP connector configuration for an Active Directory Lightweight Directory Services (AD LDS) server.


or use [PowerShell Connector](https://github.com/OpenIdentityPlatform/OpenICF.Net/wiki/PowerShell-Connector)

**Comment by vharseko** [comment]
[Active Directory Identity Management](https://github.com/OpenIdentityPlatform/OpenIDM/wiki/Active-Directory-Identity-Management)
